core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -282,6 +282,10 @@
     private static final Optional<String> HEADER_VALUE_CACHE_SPEC =
             caffeineSpec("headerValueCache", DEFAULT_HEADER_VALUE_CACHE_SPEC);
 
+    private static final String DEFAULT_HTTP_FILE_SERVICE_CACHE_SPEC = "maximumSize=1024";
+    private static final Optional<String> HTTP_FILE_SERVICE_CACHE_SPEC =
+            caffeineSpec("httpFileServiceCache", DEFAULT_HTTP_FILE_SERVICE_CACHE_SPEC);
```

Comment:
`httpFile` -> `file`, `HTTP_FILE` -> `FILE` (I think we're gonna rename `HttpFileService` to `FileService` eventually)